### PR TITLE
[s] Flavor Text for April Fools

### PR DESCRIPTION
**Pull request by TrustyGun**
:cl:
add: Added flavor text!
add: Because we have no confidence in your creativity, they are generated in order to accurately fit your character's personality.
/:cl:

Time to go full Baystation.

### CODE REVIEW

File: code/modules/mob/living/flavortext.dm [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+		"succesfully seducing people",
+		"shitposting",
+		"bursting into song",
+		"calling the shuttle early",
+		"beating the clown",
+		"throwing Ian into the washer",
+		"playing obscure games about living on a space station",  //me irl
+		"playing poker in maintenance",
+		"wasting their life",
+		"pondering the existence of a caring god,
```
Comment (Haevacht) [inline code/modules/mob/living/flavortext.dm]: Missed a "

Comment (TrustyGun) [inline code/modules/mob/living/flavortext.dm]: woops

File: code/modules/mob/living/flavortext.dm [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+		"sexily",
+		"kindly",
+		"delicately",
+		"abruptly",
+		"brutally",
+		"savagely",
+		"creepily",
+		"bombaciously",
+		"ackwardly",
+		"mathametically",
```
Comment (uraniummeltdown) [inline code/modules/mob/living/flavortext.dm]: should be mathematically

File: code/modules/mob/living/flavortext.dm [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+	)
+	var/hobbies = list(
+		"reading books",
+		"breaking the law",
+		"killing people",
+		"pissing off people",
+		"making food",
+		"writhing around in their own vomit and filth",
+		"unsuccesfully seducing people",
+		"succesfully seducing people",
```
Comment (uraniummeltdown) [inline code/modules/mob/living/flavortext.dm]: unsuccessfully, successfully

**Comment by KorPhaeron** [comment]
PRs dont need to be open to be testmerged

**Comment by Xhuis** [comment]
Imo we should keep them open for input from more people, as they probably won't check the closed section. We can close them when the time comes, but not having proper input is exactly why last April Fools' was so terrible.

**Comment by PKPenguin321** [comment]
We need an April Fools tag or something to keep track of all of these

File: code/modules/mob/living/flavortext.dm [hunk 4]
```diff
@@ -0,0 +1,111 @@
+/mob/living
+	var/adjectives = list(
```
Comment (octareenroon91) [inline code/modules/mob/living/flavortext.dm]: You're giving EVERY LIVING MOB a DISTINCT copy of this list?

Maybe you should go `static` or something

Comment (optimumtact) [inline code/modules/mob/living/flavortext.dm]: classic meme

Comment (Cyberboss) [inline code/modules/mob/living/flavortext.dm]: Also, Put this in living_defines.dm dammit!

**Pull request closed without merging**